src/phonenumber.js

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const PHONE_NUMBER_REGEXP = /^[0-9 -\.]+$/;
+const PHONE_NUMBER_REGEXP = /^[0-9 -.]+$/;
```

Comment:
Also seems a bit weird... maybe

Reply:
These also seem to be the same https://regex101.com/r/1M2H7B/3